src/Export/Exporter.php

```diff
@@ -5,29 +5,65 @@
 namespace Omikron\FactFinder\Oxid\Export;
 
 use Omikron\FactFinder\Oxid\Export\Entity\DataProviderInterface;
+use Omikron\FactFinder\Oxid\Export\Entity\ExportEntityInterface;
 use Omikron\FactFinder\Oxid\Export\Filter\TextFilter;
 use Omikron\FactFinder\Oxid\Export\Stream\StreamInterface;
+use Omikron\FactFinder\Oxid\Utilities\FfLogger;
+use OxidEsales\Eshop\Core\Registry;
 
 class Exporter implements ExporterInterface
 {
     /** @var TextFilter */
     private $filter;
 
+    /** @var bool */
+    private $proceedWhileError;
+
+    /**
+     * @var FfLogger
```

Comment:
docblock should be of the same format as rest of the fields

Reply:
solved